code/src/components/Detail.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+import Icon from "./arrow.png";
+
+export const Detail = () => {
+    const apiKey = "3c9451d52520351f0a27222d2fb5cbb2"
+    const { movieId } = useParams()
+    const [movies, setMovies] = useState([])
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=${apiKey}&language=en-US&page=1`)
+            .then((res) => res.json())
+            .then((json) => {
+                setMovies(json)
+            })
+    }, [movieId])
+
+    return (
+
+        <section className="movie-backdrop" style={{ backgroundImage: `url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` }}>
+
+            <div className="movie-content">
+                <div className="back-movies">
+                    <Link to="/" className="link">
+                        <img src={Icon} style={{ width: "40px" }} alt="icon" />
```

Comment:
Why you added this style inside `style` attribute and not as a CSS class? Needs to be fixed